pkg/util/clusterdata/worker_profile.go

```diff
@@ -102,10 +102,11 @@ func (ce machineClientEnricher) Enrich(
 	oc.Lock.Lock()
 	defer oc.Lock.Unlock()
 
-	oc.Properties.WorkerProfiles = workerProfiles
+	oc.Properties.WorkerProfilesStatus = workerProfiles
 	return nil
 }
 
 func (ce machineClientEnricher) SetDefaults(oc *api.OpenShiftCluster) {
 	oc.Properties.WorkerProfiles = nil
```

Comment:
we need to remove this now right?  We don't want to nil this out.

Reply:
```
	oc.Properties.WorkerProfiles = nil
```

specifically the above field.  It's not actually called anywhere, but we should be careful not to have it nil out worker profiles by accident.  We should retain any user-provided initial data.